src/applications/gi-sandbox/components/profile/EstimatedBenefits.jsx

```diff
@@ -84,6 +91,10 @@ export default function EstimatedBenefits({ profile, outputs, calculator }) {
   const perTermSections = () => {
     const { perTerm } = outputs;
 
+    if (isOJT) {
```

Comment:
shouldn't this be marked as `visible: false` in `selector/calculator.js` ?

Reply:
For this one, we still want the book stipend section present on Non-OJT institutions.

Reply:
Correct, but each `calculatedBenefits.outputs.perTerm` object has a visible value and this should be set to false in `selectors/calculator` if OJT instead of being removed here

Reply:
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi-sandbox/selectors/calculator.js#L1201